Formula/metabat.rb

```diff
@@ -0,0 +1,69 @@
+class Metabat < Formula
+  # cite Kang_2015: "https://doi.org/10.7717/peerj.1165"
+  desc "Statistical framework for reconstructing genomes from metagenomic data"
+  homepage "https://bitbucket.org/berkeleylab/metabat/"
+  url "https://bitbucket.org/berkeleylab/metabat/get/b3bd9c2.tar.gz"
+  version "2.13-b3bd9c2"
+  sha256 "4b3e59c28875ee0002164705a1c91d8643b904d0ee7df16e3c92ef9c55b6b16c"
+  head "https://bitbucket.org/berkeleylab/metabat.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+
+  if OS.mac?
+    fails_with :clang
```

Comment:
This failure should probably be outside the `if OS.mac?` block. If it's because it requires OpenMP, I like adding a comment.
```ruby
    fails_with :clang # needs openmp
```